result/timelineGenerator.js

```diff
@@ -3,6 +3,13 @@ const scriptContainer = document.querySelector(".timelineScript");
 const timelineDiv = document.createElement("div");
 timelineDiv.classList.add("timeline")
 
+var modalTemp
+var boolModalTemp
+
+var height = data.eventSpace
```

Comment:
const instead of var maybe? or let if you change it somewhere, which I don't see right now